README.md

```diff
@@ -2,15 +2,21 @@
 
 [![Travis-CI Build Status](https://travis-ci.org/ropenscilabs/tic.svg?branch=master)](https://travis-ci.org/ropenscilabs/tic)
 
-The goal of tic is to facilitate testing and deployment tasks for R packages in [Travis CI](https://travis-ci.org), [AppVeyor](https://www.appveyor.com/), or the CI tool of your choice.
-The intended usage is as follows:
-- You specify the steps to be run at each stage in a central location in a simple domain-specific language
-- You add boilerplate code for installation of tic, and three function calls into tic, to [`.travis.yml`](#travis)/`appveyor.yml`/... (shown below)
-- tic takes care of checking if a step is supposed to run, preparing the step if necessary (e.g., installation of dependencies), and running the enabled steps at the right time
+The goal of tic is to facilitate testing and deployment tasks for R projects in [Travis CI](https://travis-ci.org), [AppVeyor](https://www.appveyor.com/), or the CI tool of your choice.
+
+In a nutshell, `tic` does the following:  
```

Comment:
Can we frame this using a before-after comparison? Before: logic encoded in `.travis.yml` and (differently) in `appveyor.yml`, plus some Travis CI magic in `r.rb` -- after: logic encoded in `tic.R`, boilerplate `.yml` files, `r.rb` functionality restricted to installing R and caching packages?

Would a drawing help, perhaps one like the first on http://ctlente.com/en/purrr-magic/ ?

This also allows us to mention early that users need to get familiar with CI first. Is there a text/site we can link to, that doesn't focus on using R as a language but introduces general concepts.